nimble/controller/src/ble_ll_hci_supp_cmd.c

```diff
@@ -0,0 +1,328 @@
+#endif
+#if MYNEWT_VAL(BLE_LL_ROLE_CENTRAL)
+    BIT(7) /* HCI LE Extended Create Connection */
+#endif
+#endif
+);
+
+static const uint8_t octet_38 = OCTET(
+#if MYNEWT_VAL(BLE_LL_CFG_FEAT_LL_PERIODIC_ADV) && MYNEWT_VAL(BLE_LL_ROLE_OBSERVER)
+    BIT(0) /* HCI LE Periodic Advertising Create Sync */
+    BIT(1) /* HCI LE Periodic Advertising Create Sync Cancel */
+    BIT(2) /* HCI LE Periodic Advertising Terminate Sync */
+    BIT(3) /* HCI LE Add Device To Periodic Advertiser List */
+    BIT(4) /* HCI LE Remove Device From Periodic Advertiser List */
+    BIT(5) /* HCI LE Clear Periodic Advertiser List */
+    BIT(6) /* HCI LE Read Periodic Advertiser List Size */
+#endif
+    BIT(7) /* HCI LE Read Transmit Power */
+);
+
+static const uint8_t octet_39 = OCTET(
+    BIT(0) /* HCI LE Read RF Path Compensation */
+    BIT(1) /* HCI LE Write RF Path Compensation */
+#if MYNEWT_VAL(BLE_LL_CFG_FEAT_LL_PRIVACY)
+    BIT(2) /* HCI LE Set Privacy Mode */
+#endif
+);
+
+static const uint8_t octet_40 = OCTET(
+#if MYNEWT_VAL(BLE_VERSION) >= 51 && MYNEWT_VAL(BLE_LL_CFG_FEAT_LL_PERIODIC_ADV)
+#if MYNEWT_VAL(BLE_LL_ROLE_OBSERVER)
+    BIT(5) /* HCI LE Set Periodic Advertising Receive Enable */
+#endif
+#if MYNEWT_VAL(BLE_LL_CFG_FEAT_LL_PERIODIC_ADV_SYNC_TRANSFER)
+#if MYNEWT_VAL(BLE_LL_ROLE_OBSERVER)
+    BIT(6) /* HCI LE Periodic Advertising Sync Transfer */
+#endif
+#if MYNEWT_VAL(BLE_LL_ROLE_BROADCASTER)
+    BIT(7) /* HCI LE Periodic Advertising Set Info Transfer */
+#endif
+#endif
+#endif
+);
+
+static const uint8_t octet_41 = OCTET(
+#if MYNEWT_VAL(BLE_LL_CFG_FEAT_LL_PERIODIC_ADV_SYNC_TRANSFER)
+    BIT(0) /* HCI LE Set Periodic Advertising Sync Transfer Parameters */
+    BIT(1) /* HCI LE Set Default Periodic Advertising Sync Transfer Parameters */
+#endif
+);
+
+static const uint8_t octet_43 = OCTET(
+#if MYNEWT_VAL(BLE_LL_CFG_FEAT_LL_SCA_UPDATE)
+    BIT(2) /* HCI LE Request Peer SCA */
+#endif
+);
+
+static const uint8_t octet_44 = OCTET(
+#if MYNEWT_VAL(BLE_VERSION) >= 52
+    BIT(1) /* HCI LE Set Host Feature */
+#endif
+);
+
+static const uint8_t octet_46 = OCTET(
+#if MYNEWT_VAL(BLE_LL_CFG_FEAT_LL_ENHANCED_CONN_UPDATE)
+    BIT(0) /* HCI LE Set Default Subrate */
+    BIT(1) /* HCI LE Subrate Request */
+#endif
+);
+
+static const uint8_t g_ble_ll_hci_supp_cmds[64] = {
+    octet_0,
+    0,
+    octet_2,
+    0,
+    0,
+    octet_5,
+    0,
+    0,
+    0,
+    0,
+    octet_10,
+    0,
+    0,
+    0,
+    octet_14,
+    octet_15,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    octet_25,
+    octet_26,
+    octet_27,
+    octet_28,
+    0,
+    0,
+    0,
+    0,
+    octet_33,
+    octet_34,
+    octet_35,
+    octet_36,
+    octet_37,
+    octet_38,
+    octet_39,
+    octet_40,
+    octet_41,
+    0,
+    octet_43,
+    octet_44,
+    0,
+    octet_46,
+};
+
+void
+ble_ll_hci_supp_cmd_get(uint8_t *buf)
+{
+    memcpy(buf, g_ble_ll_hci_supp_cmds, sizeof(g_ble_ll_hci_supp_cmds));
```

Comment:
memset of remaining byes is not needed anymore?

Reply:
g_ble_ll_hci_supp_cmds array is extended to 64 bytes so it's already padded with zeroes